src/Features/LanguageServer/ProtocolUnitTests/Ordering/RequestOrderingTests.cs

```diff
@@ -165,32 +169,10 @@ public async Task ThrowingMutableTaskDoesntBringDownQueue()
             // first task should fail
             await Assert.ThrowsAsync<InvalidOperationException>(async () => await waitables[0]);
 
-            // remaining tasks should have executed normally
-            var responses = await Task.WhenAll(waitables.Skip(1));
-
-            Assert.Empty(responses.Where(r => r.StartTime == default));
-            Assert.All(responses, r => Assert.True(r.EndTime > r.StartTime));
-        }
-
-        [Fact]
-        public async Task ThrowingMutableTaskDoesntMutateTheSolution()
-        {
-            var requests = new[] {
-                new TestRequest(NonMutatingRequestHandler.MethodName),
-                new TestRequest(FailingMutatingRequestHandler.MethodName),
-                new TestRequest(NonMutatingRequestHandler.MethodName),
-            };
-
-            var waitables = StartTestRun(requests);
-
-            // second task should have failed
-            await Assert.ThrowsAsync<InvalidOperationException>(async () => await waitables[1]);
-
-            var responses = await Task.WhenAll(waitables.Where(t => !t.IsFaulted));
+            // remaining tasks should be canceled
+            await Assert.ThrowsAsync<TaskCanceledException>(async () => await Task.WhenAll(waitables.Skip(1)));
```

Comment:
```suggestion
            await Assert.ThrowsAsync<TaskCanceledException>(Task.WhenAll(waitables.Skip(1)));
```

Does that work? The async to just await something else looks a bit odd otherwise.